x/market/keeper/deal_test.go

```diff
@@ -104,6 +106,36 @@ func (suite *dealTestSuite) TestGetBalanceOfDeal() {
 	suite.Require().Equal(ownerBalance, sdk.NewCoin("umed", sdk.NewInt(10000000000)).Sub(balance))
 }
 
+//TODO: Get Num of DataCertificate stored in KV store.
+func (suite *dealTestSuite) TestSellOwnData() {
+	err := suite.BankKeeper.AddCoins(suite.Ctx, acc1, defaultFunds)
+	suite.Require().NoError(err)
+
+	err = suite.BankKeeper.AddCoins(suite.Ctx, acc3, zeroFunds)
+	suite.Require().NoError(err)
+
+	tempDeal := types.Deal{
+		DataSchema:            []string{acc1.String()},
```

Comment:
For future works, we have to use valid data schema for this.
Also, there need some fail-cases for selling data (e.g. invalid schema, inactive deal, and so on).
If then, let's make some TODO for this! I want @youngjoon-lee and @gyuguen to comment for this!

Reply:
Yes, I agree with your review. I didn't write test code for fail cases yet. I will also write test code for fail cases!. Thanks you!

Reply:
i agree that we need to have tests for various corner cases. But, if you feel this PR is getting bloated, you can just leave TODO comments and do it as a next PR.

Reply:
I agree too. 
Test code is important not only for successful test cases, but also for failing cases.
I told @inchori that it would be good to test the success cases and some important failure cases first.